HISTORY.md

```diff
@@ -7,6 +7,7 @@
 * For users of dictionary compression with ZSTD v0.7.0+, we now reuse the same digested dictionary when compressing each of an SST file's data blocks for faster compression speeds.
 * For all users of dictionary compression who set `cache_index_and_filter_blocks == true`, we now store dictionary data used for decompression in the block cache for better control over memory usage. For users of ZSTD v1.1.4+ who compile with -DZSTD_STATIC_LINKING_ONLY, this includes a digested dictionary, which is used to increase decompression speed.
 * Add support for block checksums verification for external SST files before ingestion.
+* Stop writing global sequence number for external SST files unless the user explicitly says otherwise.
```

Comment:
Please explicitly say that DB with bulkloaded data shouldn't be opened by RocksDB older than which version.